File: semester5/MAP/CRUD_Project/native/app/src/main/java/ro/korina/physicalactivity/domain/Activity.kt
package ro.korina.physicalactivity.domain

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Activity (
    @SerializedName("id")
    val id: Int?,
    @SerializedName("title")
    val title: String,
    @SerializedName("type")
    val type: String,
    @SerializedName("date")
    val date: String,
    @SerializedName("startTime")
    val start_time: String = "",
    @SerializedName("duration")
    val duration: String = "",
    @SerializedName("intensity")
    val intensity: String = "",
    @SerializedName("distance")
    val distance: String = "-",
    @SerializedName("description")
    val description: String = ""
) :Parcelable

File: semester5/MAP/CRUD_Project/non-native/physical_activity/lib/domain/activity.dart
import 'dart:core';
import 'package:dataclass/dataclass.dart';

@dataClass
class Activity {
  int id;
  String title;
  String type;
  String date;
  String startTime = "";
  String duration = "";
  String intensity = "";
  String distance = "";
  String description = "";

  Activity(this.id, this.title, this.type, this.date, this.startTime, this.duration, this.intensity, this.distance, this.description);
}